maltoolbox/language/languagegraph.py

```diff
@@ -256,14 +338,23 @@ class LanguageGraphAttackStep:
     type: str
     asset: LanguageGraphAsset
     ttc: dict = field(default_factory = lambda: {})
+    overrides: bool = False
     children: dict = field(default_factory = lambda: {})
     parents: dict = field(default_factory = lambda: {})
-    description: dict = field(default_factory = lambda: {})
-    attributes: Optional[dict] = None
+    info: dict = field(default_factory = lambda: {})
+    inherits: Optional[LanguageGraphAttackStep] = None
```

Comment:
This contradicts lines 45-47 that say:

```
    # MAL languages currently do not support multiple inheritance, but this is
    # futureproofing at its most hopeful.
    super_assets: list[LanguageGraphAsset] = field(default_factory=lambda: [])
```

Super assets is a list to allow for multiple asset parents, but step.inherits points only to one of them (the first and only). Given that we don't have semantics for multiple inheritance in MAL, trying to solve this here is futile, but the contradiction in the code remains. I suggest that any attempt to cater for multiple inheritance is dropped from the code, i.e. `super_assets` becomes `super_asset`.

Reply:
This is probably a good idea in hindsight. A lot of the code is multiple inheritance compatible, but if we do decide to implement multiple inheritance the more complex parts would need to be discussed. For example, how do you handle the situation where multiple parents have the same attack step name. And until we have an answer to these questions it's not really helpful to just have the code be compatible for future features that might not even come. Even though I still think multiple inheritance might be desirable.